res/values/strings.xml

```diff
@@ -87,6 +87,8 @@
     <string name="this_month">This month</string>
     <string name="last_week">Last week</string>
     <string name="last_month">Last month</string>
+    <!-- Placeholder replaced by relative time, resulting in "Last seen 3min ago" -->
+    <string name="last_seen">Last seen %1$s</string>
```

Comment:
where does the "ago" come from? there seem to be some strings missing: i am pretty sure, this will be hard to translate to some languages, even without. for now, maybe better do what android/ios are doing and use an absolute time and  `last_seen_at` and `last_seen_unknown`

Reply:
Momentjs: https://momentjs.com/docs/#/customization/relative-time/

`Last seen vor 36 Minuten` if set to german, so should be translated to "Zuletzt gesehen"

not using relative time makes the thing much less useful, maybe we could add a collon (`:`) to make it more clear?
"Last seen: 3min ago", "Last seen: now", "Last seen: 3hours ago"

If there are languages we moments doesn't support we already have the same problem in the relative timestamps in the messages anyway.

Reply:
i see, the string "ago" comes from some dependency.

to make that more clear, maybe rename the key to `last_seen_rel`, that also matches nicely to `last_seen_at`